config/settings/base.py

```diff
@@ -296,9 +296,9 @@
 # https://django-allauth.readthedocs.io/en/latest/configuration.html
 ACCOUNT_EMAIL_VERIFICATION = "mandatory"
 # https://django-allauth.readthedocs.io/en/latest/configuration.html
-ACCOUNT_ADAPTER = "jukebox_radio.users.adapters.AccountAdapter"
+# ACCOUNT_ADAPTER = "jukebox_radio.users.adapters.AccountAdapter"
```

Comment:
Removing. More info in wiki.

https://github.com/0x213F/jukebox-radio-django/wiki/Authentication